config.w32

```diff
@@ -177,6 +177,7 @@ if (PHP_MONGODB != "no") {
     // TODO: Support building with Secure Channel on Windows
     MONGOC_ENABLE_SSL_SECURE_CHANNEL: 0,
     MONGOC_ENABLE_CRYPTO_CNG: 0,
+    MONGOC_HAVE_BCRYPT_PBKDF2: 0,
```

Comment:
Looking at CMakeLists for libmongoc, this is only used on Windows when MONGOC_ENABLE_CRYPTO_CNG is used. Since we don't support building with secure channel on Windows, we can leave this disabled without checking for it: https://github.com/mongodb/mongo-c-driver/pull/1684/files#diff-e0da4e929ea7d93528029a4c5d5bc77c7c7218fa6328628c100c8c7dc2ed5ffaR536